real_check/test/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+#Copyright (c) Microsoft. All rights reserved.
+#Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+add_subdirectory(reals)
+add_subdirectory(both)
+set(REAL_CHECK_TEST_INC_FOLDER ${CMAKE_CURRENT_LIST_DIR}/inc CACHE INTERNAL "this is what needs to be included if using sharedLib lib" FORCE)
```

Comment:
>set(REAL_CHECK_TEST_INC_FOLDER ${CMAKE_CURRENT_LIST_DIR}/inc C [](http://example.com/codeflow?start=0&length=62)

Do we need this as a cmake variable? I think not.
Having cmake variables defined like this is useful when the variable is used by other projects.

Reply:
Probably here you simply want an include_directories?

Reply:
I agree we don't need the cmake variable. I've removed it in the next iteration. The project seems to build without an `include_directories`.